Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1129 @@
+                            }
+                        }
+
+                        //determine if the param is Flow Control Param or not based on it value and agreed sign "$$_"
+                        if (paramSelectedValue.StartsWith("$$_"))
+                        {
+                            isflowControlParam = false;
+                            if (paramSelectedValue.StartsWith("$$_"))
+                                paramSelectedValue = paramSelectedValue.Substring(3);//get value without "$$_"
+                        }
+                        else if (paramSelectedValue != "<Empty>")
+                            isflowControlParam = true;
+
+                        //check if already exist param with that name
+                        VariableBase stepActivityVar = stepActivity.Variables.Where(x => x.Name.ToUpper() == param.ToUpper()).FirstOrDefault();
+                        if (stepActivityVar == null)
+                        {
+                            //#Param not exist so add it
+                            if (isflowControlParam == true)
+                            {
+                                //add it as selection list param                               
+                                stepActivityVar = new VariableSelectionList();
+                                stepActivityVar.Name = param;
+                                stepActivity.AddVariable(stepActivityVar);
+                                stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because new flow control param was added
+                            }
+                            else
+                            {
+                                //add as String param
+                                stepActivityVar = new VariableString();
+                                stepActivityVar.Name = param;
+                                ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
+                                stepActivity.AddVariable(stepActivityVar);
+                            }
+                        }
+                        else
+                        {
+                            //#param exist
+                            if (isflowControlParam == true)
+                            {
+                                if (!(stepActivityVar is VariableSelectionList))
+                                {
+                                    //flow control param must be Selection List so transform it
+                                    stepActivity.Variables.Remove(stepActivityVar);
+                                    stepActivityVar = new VariableSelectionList();
+                                    stepActivityVar.Name = param;
+                                    stepActivity.AddVariable(stepActivityVar);
+                                    stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because flow control param was added
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)